downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md

```diff
@@ -88,6 +94,10 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 10 (2201.10.0)
 - Introduced support for parsing XML with record types with default values as the expected type, using the default values where required (i.e., if a value corresponding to the record field is not present in the XML value).
 - Introduced the option to choose between semantic and syntactic equality of XML elements and attributes.
 
+#### `java.jdbc` package
+
+- Updated the `datasourceName` and `properties` field in the `jdbc:Options` record from optional value types to optional fields to allow users to use the record as a configurable variable.
```

Comment:
```suggestion
- Updated the `datasourceName` and `properties` fields in the `jdbc:Options` record from optional value types to optional fields to allow users to use the record as a configurable variable.
```

Both should have worked ideally? Was this due to a bug?

Also this is a breaking change?

Reply:
if there are optional value fields in a record, we can't make the record configurable.  I thought it was not supported. Hence made this change. 
Yes, this is a breaking change, I made it as minor release in update 10

Reply:
Seems like this is not yet supported when the type is a union type.